packages/react-select/src/Select.tsx

```diff
@@ -1724,9 +1727,10 @@ export default class Select<
       'aria-labelledby': this.props['aria-labelledby'],
       'aria-required': required,
       role: 'combobox',
-      'aria-activedescendant': this.isAppleDevice
-        ? undefined
-        : this.state.focusedOptionId || '',
+      'aria-activedescendant':
+        this.isAppleDevice && this.state.componentHasMounted
+          ? undefined
+          : this.state.focusedOptionId || '',
```

Comment:
this empty string needs to be "undefined" too for handling server side rendering